test/bench/pick-layers.bench.js

```diff
@@ -0,0 +1,74 @@
+import {Layer} from 'deck.gl/core';
+import {getClosestFromPickingBuffer} from 'deck.gl/core/lib/pick-layers';
+
+const SAMPLE_LAYERS = [new Layer()];
+const OBJECT_COLOR = [0, 10, 20, 1];
+const NULL_COLOR = [0, 0, 0, 0];
+
+const TEST_CASES = [
+  {
+    title: 'Solid',
+    data: generateSampleData({
+      pickingRadius: 10,
+      getColor: () => OBJECT_COLOR
+    })
+  },
+  {
+    title: 'Circle',
+    data: generateSampleData({
+      pickingRadius: 10,
+      getColor: (x, y) => (10 - x) * (10 - x) + (10 - y) * (10 - y) < 25 ? OBJECT_COLOR : NULL_COLOR
```

Comment:
Neat!

Maybe use some constants or add comments making this a little easier to read.

e.g. `// radius 5 circle centered in 20x20 rect.`